### ISSUE TITLE
cudaStreamCaptureModeGlobal not declared

### ISSUE THREAD

**Issue by leerumor** [NVIDIA/tensorrt-inference-server#635]
I got this error when building with CMake:
I set TensorRT and Tensorflow backend

```
tensorrt-inference-server/src/backends/tensorrt/plan_backend.cc: In member function ‘bool nvidia::inferenceserver::PlanBackend::Context::BuildCudaGraph(int)’:
tensorrt-inference-server/src/backends/tensorrt/plan_backend.cc:591:43: error: ‘cudaStreamCaptureModeGlobal’ was not declared in this scope
   cuerr = cudaStreamBeginCapture(stream_, cudaStreamCaptureModeGlobal);
```

I tried: 
1. add #include <cuda_runtime.h> and  #include <cuda_runtime_api.h> 
2. use cudaStreamCaptureMode::cudaStreamCaptureModeGlobal instead of cudaStreamCaptureModeGlobal, then got cudaStreamCaptureMode not decalred...

**Comment by deadeyegoodwin** [comment]
Is your CUDA version before 10.1? That CUDA API was added in 10.1. It is fixed in master with #575. The fix will be in the 19.09 release.

**Comment by leerumor** [comment]
> Is your CUDA version before 10.1? That CUDA API was added in 10.1. It is fixed in master with #575. The fix will be in the 19.09 release.

Thank you, my CUDA version is 10.0.130.
I will try r19.09 later

**Comment by lyxcliang** [comment]
my CUDA version is 10.0.130 too, i delete the 'cudaStreamCaptureModeGlobal', like this 'cuerr = cudaStreamBeginCapture(stream_);', then remake, and not report error. I want to know is that OK?
